Hamburg/Stabi/SHOr404.xml

```diff
@@ -230,7 +233,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
 
                     <history>
                       <origin>
-                          <origDate notBefore="1651" notAfter="1700"/>
+                        <origDate notBefore="1651" notAfter="1700">21 September 1667 A.D.</origDate>
```

Comment:
If the date can be determined precisely, why is it not given as when= instead of notBefore=/notAfter= ?